content/300-accelerate/600-faq.mdx

```diff
@@ -8,6 +8,18 @@ toc: true
 
 <TopBlock>Below are frequently asked questions about Accelerate.</TopBlock>
 
+## When do I have to enable static IP for Prisma Accelerate?
+
+A static IP address is an IPv4 or an IPv6 address that doesn’t change. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses are predictable.
+
+![What is a static IP](./images/static-ip.png)
+
+For database connections using firewalls or IP allowlisting, you need static IPs because they're reliable and consistent. After you opt-in to use Accelerate with static IP enabled, add the IP address you receive in the database firewall IP allowlist or permit it access to your private network. This ensures that Accelerate can securely access your database while other unauthorized requests to your database are denied.
```

Comment:
Is the connection itself using firewalls and IP allowlisting? I think that's rather the database itself. Would suggest rephrasing.

> because they're reliable and consistent
It's not the reliability, but the predictability. This sounds a little fluffy, especially for the docs let's be more clear and accurate.

Reply:
Great catch 🙌 , rephrased and added a few for words to it for more clarity.